pages/osx/split.md

```diff
@@ -6,9 +6,9 @@
 
 `split -l {{10}} {{filename}}`
 
-- Split a file into 5 files. File is split such that each split has same size (except the last split):
+- Split a file by a pattern (the pattern is the head of each split):
```

Comment:
Is this pattern a regular expression?

Also, any reason for removing this example and adding a new one? We haven't reached the limit for this page yet.

Reply:
at least on my macos, `-n` is not an option for split.

pattern is interpreted as a regular expression, though the manual does not specify which kind

Reply:
Ah, I see! This is the macos-specific version of the page. Got it!